checkstyle/src/main/kotlin/org/projectnessie/buildtools/checkstyle/CheckstyleHelperPlugin.kt

```diff
@@ -52,7 +51,7 @@ class CheckstyleHelperPlugin : Plugin<Project> {
             "checkstyleTest" -> dependsOn(tasks.named("processTestJandexIndex"))
             else -> {}
           }
-          if (plugins.findPlugin(QuarkusPlugin::class.java) != null) {
+          if (plugins.hasPlugin("io.quarkus")) {
```

Comment:
is this the full id or does this method do a prefix match?

Reply:
See javadoc: `... for a plugin with given id. `